vendor/assets/stylesheets/dvl/components/page_header.scss

```diff
@@ -33,35 +36,92 @@
   }
 
   > .button,
-  > form .button {
+  > form .button,
```

Comment:
If you know it's here for a reason that's fine... I'm just curious when there would be a button inside of a page header that we _don't_ want to style.